### ISSUE TITLE
Update machine image in use

### ISSUE THREAD

**Issue by badboy** [mozilla/burnham#373]
I got notified that CircleCI is removing Ubuntu 14.04 images
See https://circleci.com/docs/2.0/images/linux-vm/14.04-to-20.04-migration/ for all details.

I don't see `machine: true` in this repo's config.yaml, so I think it's because the Orb in use ([gcp-gcr](https://circleci.com/developer/orbs/orb/circleci/gcp-gcr)) is still using `machine: true` ([see code](https://github.com/CircleCI-Public/gcp-gcr-orb/blob/463e12f88d3aea70cfeef84aa7a9d010bd7422ea/src/executors/default.yml#L3)).
I hope they'll publish a new release soon then.

**Comment by hackebrot** [comment]
We currently use `circleci/gcp-gcr@0.13.0`. Do you think upgrading to `0.14.1` would solve this issue?

See https://github.com/mozilla/burnham/blob/main/.circleci/config.yml#L5

**Comment by hackebrot** [comment]
We're using the same version as https://github.com/mozilla/bigquery-etl. 🤖

**Comment by badboy** [comment]
> We currently use `circleci/gcp-gcr@0.13.0`. Do you think upgrading to `0.14.1` would solve this issue?
> 
> See [`main`/.circleci/config.yml#L5](https://github.com/mozilla/burnham/blob/main/.circleci/config.yml#L5)

Nope. I linked to the latest version and that's still with `machine: true`, so we will have to wait a bit I guess.
